LuaContext.hpp does not compile on CentOS7 after fc2084a

fc2084a 

error:
```
ext/luawrapper/include/LuaContext.hpp:1715:5: error: expected initializer before ‘LuaMetatable’
     LuaMetatable {};
     ^
ext/luawrapper/include/LuaContext.hpp:1712:5: warning: ‘LuaEmptyArray’ defined but not used [-Wunused-variable]
     LuaEmptyArray {};
     ^
```

--- a/pdns/ext/luawrapper/include/LuaContext.hpp
+++ b/pdns/ext/luawrapper/include/LuaContext.hpp
@@ -58,7 +58,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #   include "misc/exception.hpp"
 #endif
 
-#ifdef _GNUC
+#ifdef __GNUC__
 #   define ATTR_UNUSED __attribute__((unused))
 #else
 #   define ATTR_UNUSED

--- a/pdns/ext/luawrapper/include/LuaContext.hpp
+++ b/pdns/ext/luawrapper/include/LuaContext.hpp
@@ -1711,7 +1711,7 @@ private:
 static LuaContext::EmptyArray_t ATTR_UNUSED
     LuaEmptyArray {};
 /// @deprecated
-static LuaContext::Metatable_t ATTR_UNUNSED
+static LuaContext::Metatable_t ATTR_UNUSED
     LuaMetatable {};
     
 /**************************************************/

